tests/snappi/pfc/test_pfc_pause_response_with_snappi.py

```diff
@@ -0,0 +1,161 @@
+import logging
+import pytest
+
+from files.helper import run_pfc_test
+from tests.common.helpers.assertions import pytest_assert, pytest_require
+from tests.common.fixtures.conn_graph_facts import conn_graph_facts,\
+    fanout_graph_facts
+from tests.common.snappi.snappi_fixtures import snappi_api_serv_ip, snappi_api_serv_port,\
+    snappi_api, snappi_testbed_config
+from tests.common.snappi.qos_fixtures import prio_dscp_map, all_prio_list, lossless_prio_list,\
+    lossy_prio_list
+from tests.common.reboot import reboot
+from tests.common.platform.processes_utils import wait_critical_processes
+from tests.common.utilities import wait_until
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [ pytest.mark.topology('tgen') ]
+
+def test_pfc_single_lossless_headroom(snappi_api,
+                                        snappi_testbed_config,
+                                        conn_graph_facts,
+                                        fanout_graph_facts,
+                                        duthosts,
+                                        rand_one_dut_hostname,
+                                        rand_one_dut_portname_oper_up,
+                                        enum_dut_lossless_prio,
+                                        all_prio_list,
+                                        prio_dscp_map,
+                                        pfc_pause_quanta_values):
```

Comment:
How do you get this value -  pfc_pause_quanta_values? I don't see any fixture with this name

Reply:
Check the fixture comment above

Reply:
I saw the comment on L45 but where is this fixture defined? is this an existing fixture or a new one added?